assets/data/majors.json

```diff
@@ -0,0 +1,176 @@
+[
```

Comment:
this won't capture every major out there, but captures most general major categories. users can still enter in their own unique major if they have something that doesn't fit any of these